spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -377,6 +377,27 @@ def defined_private_method
         expect(object.send(:defined_private_method)).to eq("works")
       end
 
+      it 'can be temporarily supressed' do
+        without_verifying_partial_doubles do
+          expect(object).to receive(:fictitious_method) { 'works' }
+        end
+        expect(object.fictitious_method).to eq 'works'
+
+        expect {
+          expect(object).to receive(:another_fictitious_method) { 'works' }
+        }.to raise_error RSpec::Mocks::MockExpectationError
+      end
+
+      specify 'temporarily supressing partial doubles does not affect normal verifying doubles' do
+        klass = Class.new
+        object = nil
```

Comment:
What is the purpose of `klass = Class.new`, `object = nil` and the `object =` bit below?  I don't see how these variables get used.